docs/concepts/resource-manager.md

```diff
@@ -109,6 +109,7 @@ The gardener-resource-manager can manage a resource in different modes. The supp
 - `Ignore`
     - The corresponding resource is removed from the ManagedResource status (`.status.resources`). No action is performed on the cluster - the resource is no longer "managed" (updated or deleted).
     - The primary use case is a migration of a resource from one ManagedResource to another one.
+    - If this annotation is present on the ManagedResource itself then it will be skipped entirely by the controller (no reconciliations or deletions at all). However, when the ManagedResource itself is deleted then the annotation is ignored and all resources will be deleted as usually. This feature can be helpful to temporarily patch/change resources managed as part of such ManagedResource. 
```

Comment:
Or maybe we can write 
```
However, when the ManagedResource itself is deleted (for ex- when a shoot is deleted) then the annotation is ignored and all resources will be deleted as usual.
```
```suggestion
    - If this annotation is present on the ManagedResource itself then it will be skipped entirely by the controller (no reconciliations or deletions at all). However, when the ManagedResource itself is deleted then the annotation is ignored and all resources will be deleted as usual. This feature can be helpful to temporarily patch/change resources managed as part of such ManagedResource. 
```